src/components/SideBar/index.js

```diff
@@ -17,24 +17,36 @@ const SideBar = () => (
           <SideBarWrapper>
             <ul>
               <li>
-                <img src={avatar} alt="avatar profile"/>
+                <img src={avatar} alt="avatar profile" />
                 <Link to="/profile/edit">Edit Profile</Link>
               </li>
-              <li><Link to="/reports/new">Create Daily Report</Link></li>
+              <li>
+                <Link to="/reports/new">Create Daily Report</Link>
+              </li>
+              <li>
+                <Link to="/reports">Your Daily Reports</Link>
+              </li>
 
               {roles.includes('TEAM_LEADER') && (
                 <Fragment>
-                  <li><Link to="/weekly-reports/new">Create Weekly Report</Link></li>
-                  <li><Link to="/statistic">Statistic</Link></li>
+                  <li>
+                    <Link to="/weekly-reports/new">Create Weekly Report</Link>
+                  </li>
+                  <li>
+                    <Link to="/team/reports">Team's Daily Reports</Link>
+                  </li>
```

Comment:
Some weird red high lighting caused by Github issue with JSX😢
https://stackoverflow.com/questions/30667675/what-does-a-red-text-background-mean-in-github-comparison-view